qcodes/instrument_drivers/Lakeshore/Model_325.py

```diff
@@ -472,6 +472,24 @@ def __init__(self, parent: 'Model_325', name: str, loop: int) ->None:
             get_cmd=f"RAMPST? {self._loop}"
         )
 
+        self.add_parameter(
+            "resistance",
+            get_cmd=f"HTRRES? {self._loop}",
+            set_cmd=f"HTRRES {self._loop} {{}}",
+            vals=Enum(25, 50),
+            val_mapping={
+                25: 1,
+                50: 2,
+            },
+            unit="Ohm"
+        )
+
+        self.add_parameter(
+            "heater_output",
```

Comment:
please also add a `label` and `unit`

Reply:
Label "Heater Output" and unit "%" have been added.